tests/tests.py

```diff
@@ -41,27 +58,46 @@ def main(cpu: bool = False, single_gpu: bool = False, multi_gpu: bool = False, d
 
     # run cpu / single-gpu tests
     if cpu or single_gpu:
+        print("\n=== RUN CPU & SINGLE-GPU TESTS ===" if single_gpu else "\n=== RUN CPU TESTS ===")
         command_unit_tests = (
             f"cd {_ROOT_DIR} && CUDA_VISIBLE_DEVICES={devices[0] if single_gpu else None} python -m pytest"
         )
-
-        print("\n=== RUN CPU & SINGLE-GPU TESTS ===" if single_gpu else "\n=== RUN CPU TESTS ===")
         print(command_unit_tests)
         subprocess.run(command_unit_tests, shell=True, capture_output=False, text=True)
 
     # run multi-gpu tests
     if multi_gpu:
+        # distributed tests
+        print("\n=== RUN MULTI-GPU TESTS ===")
         run_distributed_tests_directory = _ROOT_DIR / "tests"
         run_distributed_tests_script = _ROOT_DIR / "tests" / "run_distributed_tests.sh"
         assert isfile(run_distributed_tests_script), f"ERROR! {run_distributed_tests_script} does not exist."
         command_end_to_end_tests = (
             f"cd {run_distributed_tests_directory}; bash run_distributed_tests.sh {devices[0]} {devices[1]}"
         )
-
-        print("\n=== RUN MULTI-GPU TESTS ===")
         print(command_end_to_end_tests)
         subprocess.run(command_end_to_end_tests, shell=True, capture_output=False, text=True)
 
+        # getting started example
+        print("\n=== RUN GETTING STARTED EXAMPLE ===")
+        run_getting_started_example_directory = _ROOT_DIR / "examples" / "getting_started"
+        run_getting_started_example_script = (
+            _ROOT_DIR / "examples" / "getting_started" / "run_getting_started_example.sh"
+        )
+        assert isfile(
+            run_getting_started_example_script
+        ), f"ERROR! {run_getting_started_example_script} does not exist."
+        command_getting_started_example = (
+            f"cd {run_getting_started_example_directory}; bash run_getting_started_example.sh {devices[0]},{devices[1]}"
+        )
+        print(command_getting_started_example)
+        subprocess.run(command_getting_started_example, shell=True, capture_output=False, text=True)
+
+        output_directory = join(run_getting_started_example_directory, "data", "mem_map")
+        clear_getting_started_example_output_directory(output_directory)
+
```

Comment:
What about a check that the files have been created including idx, pbin and checkpoint files?